src/mame/seibu/seibucop_cmd.ipp

```diff
@@ -27,9 +27,6 @@ void raiden2cop_device::execute_0205(int offset, uint16_t data)
 	int ppos =        m_host_space->read_dword(cop_regs[0] + 0x04 + offset * 4);
 	int npos = ppos + m_host_space->read_dword(cop_regs[0] + 0x10 + offset * 4);
 	int delta = (npos >> 16) - (ppos >> 16);
-#if LOG_Move0205
-	// ...
-#endif
```

Comment:
Should there be a commented `LOGMASKED(LOG_MOVE0905, );` as a replacement placeholder?

Reply:
Possibly. Added a commented-out `LOGMASKED(LOG_MOVE0205, ...);` to reflect the comment that was there.